### PR TITLE
Get current route

### PR DESCRIPTION
**Pull request by JohannesDoberer**
added App.svelte as internal variable to Luigi

### CODE REVIEW

File: core/src/core-api/_internalLinkManager.js [hunk 1]
```diff
@@ -87,13 +88,23 @@ export class linkManager extends LuigiCoreAPIBase {
...
 
   fromVirtualTreeRoot() {
     this.options.fromContext = null;
     this.options.fromClosestContext = false;
     this.options.fromVirtualTreeRoot = true;
+    this.options.fromParent = false;
+    return this;
+  }
+
+  fromParent() {
```
Comment (ndricimrr) [inline core/src/core-api/_internalLinkManager.js:103]: Why the need to add fromParent or just to have it in sync with the rest 🤔

File: core/src/core-api/_internalLinkManager.js [hunk 2]
```diff
@@ -125,6 +136,10 @@ export class linkManager extends LuigiCoreAPIBase {
     });
   }
 
+  getCurrentRoute() {
```
Comment (ndricimrr) [inline core/src/core-api/_internalLinkManager.js:139]: would be nice to just have a small JSDoc on this fuction only. More specifically to explain how we get to the **buildPathForGetCurrentRoute** function through _app -> ctx -> props -> . Interesting syntax but cumbersome for someone reading the line for the first time.

**Review by ndricimrr — COMMENTED** [review]
Seems to be added so far for Core-Api , Core-WC-clientAPI, but not Container WC-based ClientAPI [here](https://github.com/SAP/luigi/blob/main/container/src/services/webcomponents.service.ts#L91). Do we plan to add that there too ?

File: core/src/core-api/_internalLinkManager.js (hunk 1 above)
Comment (JohannesDoberer) [reply core/src/core-api/_internalLinkManager.js:103]: Yes, to keep it in sync.
Tested different scenarios with our current functionality like:
_...fromClosestContext().getCurrentRoute()_
_...fromContext('ctx').getCurrentRoute()_
and realized that getParent().getCurrentRoute is not supported.

**Review by ndricimrr — APPROVED** [review]
Looks good

**Pull request merged**